corva/api.py

```diff
@@ -0,0 +1,114 @@
+import posixpath
+import re
+from typing import Optional
+
+import requests
+
+
+class Api:
+    """Provides a convenient way to access Corva API and Corva Data API
+
+    Api is a thin wrapper around `requests` library that adds
+     authorization, convenient url usage  and timeouts to requests.
+    """
+
+    TIMEOUT = 30  # seconds
+
+    def __init__(
+        self,
+        *,
+        api_url: str,
+        data_api_url: str,
+        api_key: str,
+        app_name: str,
+        timeout: Optional[int] = None,
+    ):
+        self.api_url = api_url
+        self.data_api_url = data_api_url
+        self.api_key = api_key
+        self.app_name = app_name
+        self.timeout = timeout or self.TIMEOUT
+
+    @property
+    def auth_headers(self):
```

Comment:
This should be renamed to `headers` or `standard_headers`.

Reply:
Can we add the current app's version here, and pass it in as `X-Corva-App-Version`?

Reply:
yes, where do I take it from

Reply:
Good question. Let me make a new task to deal with that, it's a bigger issue.